### DISCUSSION TITLE
Error when update customer data to Quickbook Online

### DISCUSSION THREAD

**Discussion by cuongtran-6618** [codatio/codat-docs#274] [Q&A] [answered]
Hi,

I hope you have a great day. I need help with this situation. I have a Codat sandbox which connects to a QBO sandbox account. I have exported the invoice to Codat successfully and created a new customer with that.

In case user changed their information from our end, we need to reflect the change to Codat also. 

However, it didn't go through and this error appear which seems not unclear to me. Could you please double-check it?

"Failed to update Customer as it has been modified since the last sync. You may need to re-sync the Customer data."

**Comment by jmyorston** [comment, marked as answer]
Hi @cuongtran-6618 

The error you described happens if the last modified date of the item in the linked platform is later than the last modified date we have for it in our database. This means that we don't have the latest version, so we could potentially overwrite changes that had been made in the platform if we allowed the update to go through.
 
If you see this error, you would need to trigger a new pull of that datatype from the linked platform, in this case the customers dataset. This can be done by calling POST /companies/{companyId}/data/queue/{dataType}.
 
Once this has completed, you would then be able to post the update.
